### PR TITLE
datashader 0.14.3

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Jira ticket:** [PKG-817](https://anaconda.atlassian.net/browse/PKG-817) (datashader-0.14.3)

**The upstream data:**
Github releases:  https://github.com/holoviz/datashader/releases
[Diff between the latest and previous upstream releases](https://github.com/holoviz/datashader/compare/v0.14.2...v0.14.3)
License: INFO: https://github.com/holoviz/datashader/blob/master/LICENSE.txt
Changelog: https://github.com/holoviz/datashader/blob/master/CHANGELOG.rst
Requirements:
 * setup.py:  https://github.com/holoviz/datashader/blob/v0.14.3/setup.py
 *  pyproject.toml:  https://github.com/holoviz/datashader/blob/v0.14.3/pyproject.toml

**_Actions:_**

1. Update pinnings in run, see https://github.com/holoviz/datashader/blob/v0.14.3/setup.py#L11-L24
2. Update test/requires
3. Add a new test command `datashader --version`

**Additional info:**

<details>

**Package's statistics**

<details>

 * Priority B | effort: easy | Category: anaconda_affiliated | subcategory:  | pkg_type: python
 * Outdated platfroms: 'linux-64', 'linux-aarch64', 'linux-ppc64le', 'osx-64', 'osx-arm64', 'win-64'
 * Latest version: 0.14.3
 * Release on PyPi:
    * version: 0.14.3
    * date: 2022-11-17T15:30:05
* [PyPi history](https://pypi.org/project/datashader/#history)
 * Popularity: 
    * 3 months downloads: 377201.0
    * All time:  444792 downloads -  datashader  0.14.3  Data visualization toolchain based on aggregating into a grid  

</details>

**Other checks:**

<details>

1. - [x] https://github.com/holoviz/datashader/tree/v0.14.3 exists
4. - [x] https://datashader.org is present
5. - [x] Check the pinnings
6. - [x] https://github.com/holoviz/datashader/blob/master/CHANGELOG.rst exists
7. - [x] Additional research
    https://github.com/holoviz/datashader/issues
    200
8. - [x] `dev_url` is present
    https://github.com/holoviz/datashader
9. - [x] `doc_url` is present
    https://datashader.org
10. - [x] Verify that the `build_number` is correct
11. - [x] has `setuptools`
12. - [x] has `wheel`
13. - [x] `pip` in test
14. - [x] Verify the test section
15. - [x] Verify if the package is `architecture specific`
16. - [x] Verify that private modules are not mentioned in the recipe For example: (_private_module)
17.  - [x] license_file: LICENSE.txt is present
18. - [x] License: BSD-3-Clause
    - [x] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier                           |
|:-------------------------------------|
| BSD-3-Clause                         |
| BSD-3-Clause-Attribution             |
| BSD-3-Clause-Clear                   |
| BSD-3-Clause-LBNL                    |
| BSD-3-Clause-Modification            |
| BSD-3-Clause-No-Military-License     |
| BSD-3-Clause-No-Nuclear-License      |
| BSD-3-Clause-No-Nuclear-License-2014 |
| BSD-3-Clause-No-Nuclear-Warranty     |
| BSD-3-Clause-Open-MPI                |

19. - [x] license_family BSD is present
</details>

**Check dependency issues:**

<details>
../aggregate/datashader-feedstock/recipe/meta.yaml
Dependencies: ['param >=1.6.1', 'colorcet >=0.9.0', 'python', 'dask-core', 'xarray >=0.9.6', 'setuptools', 'requests', 'pip', 'pyct', 'numba >=0.51', 'pyct >=0.4.5', 'pandas >=0.24.1', 'pillow >=3.1.1', 'scipy', 'datashape >=0.5.1', 'param', 'wheel']

noarch:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

linux-64:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: Encountered problems while solving:
  - nothing provides __glibc >=2.17 needed by libgcc-ng-11.2.0-h1234567_0

linux-ppc64le:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

linux-s390x:
- py3.7: The following packages are missing from the target environment:
  - datashape >=0.5.1

- py3.8: The following packages are missing from the target environment:
  - datashape >=0.5.1

- py3.9: The following packages are missing from the target environment:
  - datashape >=0.5.1

- py3.10: The following packages are missing from the target environment:
  - numba >=0.51

osx-64:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

osx-arm64:
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

win-64:
- py3.7: No issues found
- py3.8: No issues found
- py3.9: No issues found
- py3.10: No issues found

</details>

**Links:**
* [Anaconda Recipes feedstock](https://github.com/AnacondaRecipes/datashader-feedstock)
* [Update branch](https://github.com/AnacondaRecipes/datashader-feedstock/tree/0.14.3)
* [conda-forge recipe](https://github.com/{upstream}/{feedstock_name}-feedstock)
* [PyPI](https://pypi.org/project/datashader)
* [Diff between upstream and feature branch](https://github.com/conda-forge/datashader-feedstock/compare/main...AnacondaRecipes:0.14.3)
* [Diff between origin and feature branch](https://github.com/AnacondaRecipes/datashader-feedstock/compare/master...AnacondaRecipes:0.14.3)
* [The last merged PRs](https://github.com/AnacondaRecipes/datashader-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

</details>

**Updating the recipe:**
If the recipe needs additional modification the update branch can be modified. Note that the PR diffs are not updated with these changes.
```
git clone -b 0.14.3 git@github.com:AnacondaRecipes/datashader-feedstock.git
```

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,33 +28,36 @@ requirements:
     - wheel
   run:
     - python
-    - colorcet >=0.9.0
+    - colorcet
     - dask-core
-    - datashape >=0.5.1
```
Comment (katietz) [inline recipe/meta.yaml:33]: why we remove all those version constrains here?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,33 +28,36 @@ requirements:
...
+    - datashape
     - numba >=0.51
-    - pandas >=0.24.1
-    - param >=1.6.1
-    - pillow >=3.1.1
-    - pyct >=0.4.5
+    - numpy
```
Comment (katietz) [inline recipe/meta.yaml]: this looks not right, as it is just pinned to the random version from cbc.  it hould be something >=....,<2 ...

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:33]: They were removed by the maintainers https://github.com/holoviz/datashader/blob/v0.14.3/setup.py#L11-L24

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: I've pinned `numpy >=1.19,<2.0a`

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:33]: This PR curated dependencies and made them without pinnings https://github.com/holoviz/datashader/pull/1116

**Review by cbouss — APPROVED** [review]
LGTM

**Pull request merged**